tfjs-backend-webgpu/src/backend_webgpu.ts

```diff
@@ -485,21 +454,23 @@ export class WebGPUBackend extends KernelBackend {
 
     const size =
         util.sizeFromShape(shape) * webgpu_util.GPUBytesPerElement(dtype);
-    const resBuffer = this.acquireBuffer(size);
+    const buffer =
+        this.bufferManager.acquireBuffer(size, this.defaultGpuBufferUsage());
```

Comment:
nit: `const buffer = this.bufferManager.acquireBuffer(resourceInfo.size, this.defaultGpuBufferUsage());`?